api/wasm.go

```diff
@@ -562,6 +562,12 @@ type Memory interface {
 	WriteString(offset uint32, v string) bool
 }
 
+// CustomSection contains the name and raw data of a custom section.
+type CustomSection interface {
+	Name() string
+	Data() []byte
```

Comment:
In Go in general, every single one of good public API *must* have Godoc! https://go.dev/doc/comment
```suggestion
	// Name ...
	Name() string
	// Data ...
	Data() []byte
```